webapp/src/ts/services/select2-search.service.ts

```diff
@@ -195,6 +205,15 @@ export class Select2SearchService {
     }
   }
 
+  private getContactId() {
+    let activeRoute = this.route.firstChild;
+    while (activeRoute?.firstChild) {
+      activeRoute = activeRoute.firstChild;
+    }
+
+    return activeRoute?.snapshot?.params?.id;
```

Comment:
Currently this `with-same-parent` functionality does not work with `contact` forms because those have a `parent_id` value here instead of `id`.  Not sure if this was intentional, but it seems like it might be useful to have this functionality in a contact form (especially if it is pretty simple to include here in the logic...)

Reply:
Thank you! Fixed